### ISSUE TITLE
KeybaseInstaller Helper Tool -- Mojave

### ISSUE THREAD

**Issue by spidercam** [keybase/client#14183]
I keep getting a popup saying:

> 'KeybaseInstaller is trying to install a new helper tool.'

Even after entering the local machine admin password, it asks for it again over and over again 10-20 minutes between each popup.

**my log id:** c051e9df2b42cd6d439ed81c
**macOS version:** 10.14.1 Beta (18B57c)

![screen shot 2018-10-10 at 8 05 39 am](https://user-images.githubusercontent.com/113851/46746239-7b769e00-cc63-11e8-891c-f9bb54651d4c.jpg)

**Comment by alexclst** [comment]
I have the same issue on macOS 10.14.2, log id 28ca748dcb46ada13e6be51c.

**Comment by maxtaco** [comment]
Hi we are very sorry, we have a regression here. KBFS in this release only works for administrator users. We are working on a fix but it might be delayed due to the holiday weekend. The only workaround we now know of is to run locally as an administrator user.  Or not use Finder integration since all other features should work.

**Comment by bertjwregeer** [comment]
Kicker is that when you try to uninstall the fuse components, you can't as non-admin, and if you use sudo to do it as root it says this program can't run as root:

```
Riley:~ xistence$ sudo keybase --debug uninstall -c fuse
Oops, you are trying to run as an admin user (Uid: 0). This isn't supported.
```

So you have to make sure to pass an `admin` user with `sudo -u <admin username>` to allow keybase to uninstall the fuse/finder integration.

**Comment by maxtaco** [comment]
Good point we will make sure to fix that too I think the same solution suffices that we have in mind for letting authorized non-admins run KBFS

**Comment by pathoftruth** [comment]
Commenting to receive updates on when fix is published.

**Comment by scolby33** [comment]
I am having the same issue. My error dialog is a bit different though: https://i.imgur.com/PYiN6If.png

This happened repeatedly with a Homebrew install, so I tried uninstalling that and using the DMG from keybase.io but encountered the same problem.

Log ID: 57f3e41ea4bccaf861fac01c

**Comment by maxtaco** [comment]
Sorry, we had a bug. New release coming out today or tomorrow, or if you want in the mean time, you can try this preview:

https://s3.amazonaws.com/prerelease.keybase.io/darwin-test/Keybase-2.12.5-20190101163554%2Bc4dec09.dmg

**Comment by Lex007KY** [comment]
I also have this problem. Following to keep track of the resolution. I was able to install the above update correctly. We'll see if it's a temporary or long-term fix to the popups.
